gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -17,6 +17,14 @@ <h1 class="title">
           <a class="vulnerability-improvement-link" href="{{ vulnerability.human_source_link }}/improve">
             Suggest an improvement
           </a>
+        {% elif vulnerability.human_source_link -%}
+          <div class="vulnerability-improvement-link">
+          <a href="https://google.github.io/osv.dev/faq/#ive-found-something-wrong-with-the-data" title="Follow the Source link below and use its record feedback reporting mechanism">
```

Comment:
I think you need to add `target="_blank" rel="noopener noreferrer"` to this link as well since it is an external link.

Reply:
Doh, I missed this before I merged it, will send a follow-up PR.

Reply:
https://github.com/google/osv.dev/pull/2823